### PR TITLE
Fix lint issues and remove debug code

### PR DESCRIPTION
**Pull request by lscheinkman**
@marionleborgne, @brev I've cleaned up some of the old `oobe` debug code and soften a little the lint rules fixing some of the lint issues and documentation.

Please review

### CODE REVIEW

File: unicorn/js/main/index.js [hunk 1]
```diff
@@ -52,15 +51,9 @@ crashReporter.start({
 });
 
 app.on('window-all-closed', () => {
-  // OS X apps stay active until the user quits explicitly Cmd + Q
-  if (process.platform !== 'darwin') {
```
Comment (brev) [inline unicorn/js/main/index.js]: @lscheinkman u sure about this one? I am used to Mac apps having this behavior. Just double-checking.  cc @marionleborgne

**Comment by brev** [comment]
:+1:

File: unicorn/js/main/index.js (hunk 1 above)
Comment (lscheinkman) [inline unicorn/js/main/index.js]: @brev this behavior causes a lot of confusion when closing the last window. It leaves the application open, the database locked and no way to restore the main window. 
I've seen this behavior on Apple's own applications (`Facetime`, `Calendar`, `Photos`, `Messages`, ...).

Comment (brev) [inline unicorn/js/main/index.js]: @lscheinkman great thanks

**Pull request merged**